### ISSUE TITLE
Feature Idea: View all my cards from all boards

### ISSUE THREAD

**Issue by biva** [mattermost/focalboard#2150]
## Summary

Currently, I can view each single board within each single channel. And within a card, I can have a property to assign a card to somebody.

I would like to see all the cards assigned to me, within 1 single view.

## How important this is to me and why
We use cards to manage todos. This is more efficient to have 1 single view, compared to looking into each single board.

Use cases:
1. We have 2 channels: _Project_ and _Kitchen_
2. In _Project_, 2 cards (TaskA and TaskB) are assigned to me, within the property _Who_
3. In Kitchen, 1 card (TaskC) is assigned to me, within the property _Who_
4. I would love to have a view showing TaskA, TaskB and TaskC together. A simple table would be a great start. And a calendar view would be even better (not a priority though)

## Additional context/similar features
Trello does something similar, as far as I understand: https://help.trello.com/article/819-viewing-all-of-your-cards

**Comment by guy0902** [comment]
Our team work on multiple s projects simultaneously. We could also use this feature as a way to see all tasks from all projects on on page and sort by assignee, priority, or any other category defined by user. It would provide managers with an overview of all projects in one single view.

**Comment by allahcurse** [comment]
Yeah, we are a team of 4 departaments, with non related tasks with 1 manager over all four deps. Inability for the manager to view all is the only thing thats stopping us from switching over to boards
